inferno-types/src/Inferno/Types/VersionControl.hs

```diff
@@ -282,12 +301,13 @@ pinnedUnderVCToMaybe = \case
   _ -> Nothing
 
 vcObjectHashToByteString :: VCObjectHash -> ByteString
```

Comment:
We could remove `vcObjectHashToByteString` and `byteStringToVCObjectHash` but that would require to update `plow/all` code that relies on those functions too.